_config.yml

```diff
@@ -30,6 +30,8 @@ collections:
     output: true
   projects:
     output: true
+  staff:
+    output: true
```

Comment:
Not needed, we shouldn't be outputting a page for each staff member if there is no staff detail page